src/Platform.php

```diff
@@ -10,7 +10,7 @@
 class Platform
 {
     const REPOSITORY_DIR = 'repository';
-    const SHARED_DIR = 'shared';
+    const SHARED_DIR = '.platform/local/shared';
```

Comment:
This change will break some of my flows. Maybe we should check if `.platform` exists? I've started to use this tool more for non Platform.sh sites than Platform.sh sites